tests/identity.rs

```diff
@@ -21,6 +21,7 @@ fn nonblocking_identity_detection() {
         env,
         mut certify_rsp,
         client_config: _,
+        server_config: _,
```

Comment:
TIOLI: you could also just say
```rust
    let identity::Identity { 
        env, 
        mut certify_rsp, 
        ..  
    } = identity::Identity::new("foo-ns1", id.to_string());
```
which is a little bit more concise (and if we imported `Identity`, would probably fit on one line!)

(ref: https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html#ignoring-remaining-parts-of-a-value-with-)